src/main/java/bg/startit/hackathon/airquiality/validation/PasswordConstraintValidator.java

```diff
@@ -0,0 +1,53 @@
+package bg.startit.hackathon.airquiality.validation;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.passay.*;
+
+public class PasswordConstraintValidator implements ConstraintValidator<ValidPassword, String> {
```

Comment:
This is for annotations. However we can't use annotations, because the class is generated.
So we need a Validator service, that accepts ChangePasswordRequest

See https://docs.spring.io/spring-framework/docs/3.2.x/spring-framework-reference/html/validation.html

Reply:
We understood that we cannot add annotations to the request as it is generated. That is why we added a static method which we use in the controller "isPasswordStatic". We have not removed the annotation  as we can add a proxy validator but it is more work.